tutorials/understanding-group/index.md

```diff
@@ -0,0 +1,327 @@
+
+## Create group policy
+
+Next we need to decide of a group policy. This defines how long a proposal can be voted on and how the outcome is calculated.
+Here we use the `ThresholdDecisionPolicy`, it defines a threshold of yes votes (based on a tally of voter weights) that must be achieved in order for a proposal to pass.
+
+Following is the content of the `policy.json`. Proposals can be voted on for a maximum of 10 minutes.
+A proposal requires only one vote to pass.
+
+```json
+{
+    "@type": "/cosmos.group.v1.ThresholdDecisionPolicy",
+    "threshold": "1",
+    "windows": {
+        "voting_period": "10m",
+        "min_execution_period": "0s"
+    }
+}
+```
+
+
+```sh
+simd tx group create-group-policy $ALICE_KEY 1 "" policy.json
+```
+
+Let's verify our newly created group policy and save its address for future use:
+
+```sh
+simd query group group-policies-by-group 1
+export GROUP_POLICY_ADDRESS=$(simd query group group-policies-by-group 1 --output json | jq -r '.group_policies[0].address')
+```
+
+## Vote on proposal
+
+Now that we have a group with a few members and a group policy, let's submit our first group proposal.
+Like for members management, we need to create a `proposal.json` file that contains the proposal.
+
+A proposal can be of any message. For this tutorial, we continue with our example of an association.
+The treasurer, Bob, wants to send money to a third party to pay the bills and creates a `proposal.json`:
+
+
+```json
+{
+    "group_policy_address": "cosmos1..." , // the $GROUP_POLICY_ADDRESS
+    // array of proto-JSON-encoded sdk.Msgs
+    "messages": [
+        {
+            "@type": "/cosmos.bank.v1beta1.MsgSend",
+            "from_address": "cosmos1...", // the $GROUP_POLICY_ADDRESS
+            "to_address": "cosmos1zyzu35rmctfd2fqnnytthheugqs96qxsne67ad", // a third-party
+            "amount": [
+                {
+                    "denom": "stake",
+                    "amount": "100"
+                }
+            ]
+        }
+    ],
+    "metadata": "costs utilities",
+    "proposers": [ "cosmos1..." ] // $BOB_KEY
+}
+```
+
+This proposal, if pass, will send 100 stake to cosmos1zyzu35rmctfd2fqnnytthheugqs96qxsne67ad to pay the bills.
+The tokens will be sent from the decision policy address.
+
+::: tip
+The decision policy has no fund yet. We can fund it by sending a transaction with `simd tx bank send alice $GROUP_POLICY_ADDRESS 100stake`.
+:::
+
+
+```sh
+simd tx group submit-proposal proposal.json --from bob
+```
+
+## View and vote on proposals
+
+Let's see the proposal of our football association:
+
+```sh
+simd query group proposals-by-group-policy $GROUP_POLICY_ADDRESS
+```
+
+We can see our proposal has been submitted.
+
+Next, we have Alice and Bob vote `Yes` on it and verify that our two votes are tallied:
+
+```sh
+simd tx group vote 1 $ALICE_KEY VOTE_OPTION_YES "agree"
+simd tx group vote 1 $BOB_KEY VOTE_OPTION_YES "agree"
+simd query group tally-result 1
```

Comment:
```suggestion
simd tx group vote 1 $ALICE VOTE_OPTION_YES "agree"
simd tx group vote 1 $BOB VOTE_OPTION_YES "agree"
simd query group tally-result 1
```
I suppose that the tally is updated after a vote transaction has been validated. Does this mean we should submit them with `--sync-mode block` to avoid race conditions?

Reply:
Broadcast mode `block` is deprecated, we should not use it.